CMakeLists.txt

```diff
@@ -143,6 +147,7 @@ endfunction()
 openpmd_option(MPI            "Parallel, Multi-Node I/O for clusters"     AUTO)
 openpmd_option(HDF5           "HDF5 backend (.h5 files)"                  AUTO)
 openpmd_option(ADIOS2         "ADIOS2 backend (.bp files)"                AUTO)
+openpmd_option(Julia          "Enable Julia bindings"                     OFF)
```

Comment:
For conventions:
```suggestion
openpmd_option(JULIA          "Enable Julia bindings"                     OFF)
```

Reply:
Please add an entry in the two tables here:
- https://openpmd-api.readthedocs.io/en/0.15.1/dev/buildoptions.html
- https://github.com/openPMD/openPMD-api/#from-source

Reply:
Done.